analytics/service.go

```diff
@@ -42,14 +42,16 @@ func NewServiceImpl(backend ServiceBackend) *ServiceImpl {
 
 // CaptureAnalyticsData - captures the analytics values
 func (s *ServiceImpl) CaptureAnalyticsData(r *http.Request) (string, error) {
+	cfg, err := config.Get()
```

Comment:
You need to do a `if err != nil` check here. Do you need to get the config here or could you pass it in to the method?

Reply:
As per discussions, I've updated this to pass the cfg value as a string into the function. The search handler struct has been updated to accept the cfg value as a property too